.circleci/test-example-repo.sh

```diff
@@ -36,15 +36,21 @@ then
     BUILD_TOOLS_VERSION="${CIRCLE_BRANCH}#${CIRCLE_SHA1}"
 # Otherwise use the current branch
 else
-    BUILD_TOOLS_VERSION="dev-${CIRCLE_BRANCH}#${CIRCLE_SHA1}"
+    # If on root repo use the current branch
+    if [[ $CIRCLE_PROJECT_USERNAME == "pantheon-systems" ]]; then
+        BUILD_TOOLS_VERSION="dev-${CIRCLE_BRANCH}#${CIRCLE_SHA1}"
+    # Otherwise use the dev tip from the pantheon-systems repo
+    else
+        BUILD_TOOLS_VERSION="dev-master"
```

Comment:
This isn't quite right, because it means that you'll be testing the dev-master branch of the Terminus Build Tools plugin in your PR, and not your modified code. That is, the code from your PR -will- be used to create the test environment, but when the test environment runs it will use head of master instead.